plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -510,37 +517,27 @@ const attachProductAttributesTracks = () => {
 				// skip in case the button is disabled
 				return;
 			}
-			const newAttributesCount = document.querySelectorAll(
-				'.woocommerce_attribute'
+
+			const localAttributesCount = document.querySelectorAll(
+				'.woocommerce_attribute:not(.taxonomy)'
 			).length;
-			if ( newAttributesCount > attributesCount ) {
-				const local_attributes = [
-					...document.querySelectorAll(
-						'.woocommerce_attribute:not(.pa_glbattr)'
-					),
-				].map( ( attr ) => {
-					const terms =
-						(
-							attr.querySelector(
-								"[name^='attribute_values']"
-							) as HTMLTextAreaElement
-						 )?.value.split( '|' ).length ?? 0;
-					return {
-						name: (
-							attr.querySelector(
-								'[name^="attribute_names"]'
-							) as HTMLInputElement
-						 )?.value,
-						terms,
-					};
-				} );
-				recordEvent( 'product_attributes_add', {
-					page: 'product',
-					enable_archive: '',
-					default_sort_order: '',
-					local_attributes,
-				} );
-			}
+
+			const globalAttributesCount = document.querySelectorAll(
+				'.woocommerce_attribute.taxonomy'
+			).length;
+
+			recordEvent( 'product_attributes_save', {
+				page: 'product',
```

Comment:
We don't technically need this prop anymore, but we can keep it here for now, as it isn't causing any harm.